date/date.php

```diff
@@ -508,7 +508,7 @@ function gmmktime(
     ?int $month = null,
     ?int $day = null,
     ?int $year = null,
-    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $is_dst = null
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '5.6')] $is_dst = null
```

Comment:
The same as above: https://3v4l.org/nqTZf#v5523, I'd say it's `(from: '5.5', to: '5.6')`

Reply:
Fixed